### ISSUE TITLE
no results in output folder

### ISSUE THREAD

**Issue by wang-tan** [Taiji-pipeline/Taiji#36]
Hi，
I run the taiji with my ATAC-seq and RNA-seq data, there's no error and  the stdout indicates it's OK,  however, there is nothing in the output folder except a genome.index file.

![image](https://user-images.githubusercontent.com/9820714/218055047-cfef621c-d69d-4646-805c-9e605b5e7b13.png)
![image](https://user-images.githubusercontent.com/9820714/218055490-c32ae4b2-0938-4cb5-97f1-507f77da7fa0.png)

Is there something wrong? 

Best,
Tan

**Comment by wang-tan** [comment]
when I run taiji again，it only output this and nothing in output folder too.

![image](https://user-images.githubusercontent.com/9820714/218081840-1f4a7d92-de62-4ee5-95b1-b8cdd5bd8e33.png)

**Comment by kaizhang** [comment]
This is usually because your input file contains zero data, probably because of a formatting error.

**Comment by wang-tan** [comment]
Here is my input file，I really can't find anything wrong. Could you take a look for me? Thanks very much!

![image](https://user-images.githubusercontent.com/9820714/218244467-810f5274-3c4d-4d66-8735-53e787b6aaa7.png)

**Comment by AlisaOmel** [comment]
I am also getting this issue. I ran 2 different .bed files with individual input and config files. 

1) The first one finished successfully, with no output data, and a log file that matches the first comment here. 

2) Then I changed my config file and input file to a completely different set of data, and I received a log file with just:
^[[0m^[[32m^[[1m[INFO][02-21 11:32] ^[[0m^[[0m^[[32mProgram finishes successfully^[[0m

It is possible my formatting could be wrong. Could you provide an example of what my .bed file should look like/ what format I should use. I attached a copy of my .bed file here.

![Screen Shot 2023-02-21 at 11 44 27 AM](https://user-images.githubusercontent.com/23412608/220407278-3539f5d0-e66f-494f-8fd8-dc31daeb65fa.png)

**Comment by kaizhang** [comment]
@AlisaOmel The issue may relate to editing  the `input.tsv` file under Windows system which adds some special characters at the end of each line. The solution is to add a random column after the last column OR use the `yaml` format to write inputs.

You need to delete the `sciflow.db` if you want to start a fresh run.

**Comment by AlisaOmel** [comment]
Thanks so much for the quick reply @kaizhang 
Thanks for suggesting to delete sciflow.db that works. 

I edited the input.tsv file, and it did not help. I also created an input.yml file and that also did not change the output. 

Here are the 2 versions of the input files and the config file I am using. 

input_atac.tsv
<img width="1178" alt="Screen Shot 2023-02-22 at 5 41 50 PM" src="https://user-images.githubusercontent.com/23412608/220777351-695fcdf7-1c16-470e-80c4-fc3c7886d26f.png">

input_atac.yml
<img width="603" alt="Screen Shot 2023-02-22 at 5 42 19 PM" src="https://user-images.githubusercontent.com/23412608/220777420-b561dd9f-9df8-42a3-a663-4952b1456e12.png">

config_atac.yml
<img width="511" alt="Screen Shot 2023-02-22 at 5 42 56 PM" src="https://user-images.githubusercontent.com/23412608/220777497-f37426a0-70cb-48df-ba8b-46169dd70926.png">
